index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/resources/css/style.css" />
+    <title>Portfolio White</title>
+  </head>
+  <body>
+    <!-- Nav bar -->
+    <main class="main__introduction background-primary">
+      <section class="nav">
+        <nav class="nav__container">
+          <div class="nav__logo">My Logo</div>
```

Comment:
- [ ] Kindly wrap the logo in an `<a>`, it is a good practice to wrap the logo in an `<a>` as this will serve as a link to the home from anywhere on the page.